nn_surrogate_benchmark/bbob_sampler.py

```diff
@@ -27,16 +28,17 @@ def generate_lhs_samples(
 
 def main():
     BOUNDS = [-5.0, 5.0]
-    N_SAMPLES = 10_000
+    N_SAMPLES = 30_000
+    GET_GRADIENTS = True
     DIMENSIONS = 2
     INSTANCE_ID = 1
-    FUNCTION_ID = 1
+    FUNCTION_ID = 24
```

Comment:
We should refactor these variables into a configuration class or file, though it’s fine for now.